gdl_hyperopt_template.py

```diff
@@ -62,20 +61,23 @@ def objective_with_args(hparams, params, config_path):
     """
 
     # ToDo: This is dependent on the specific structure of the GDL config file
-    params['training']['target_size'] = hparams['target_size']
     params['global']['model_name'] = hparams['model_name']
-    # ToDo: Should adjust batch size as a function of model and target size...
-    params['training']['class_weights'] = [1.0, hparams['permanent_water_weight'], hparams['rivers_weight'],
```

Comment:
A (hyperopt) template should not have parameters clearly linked to a specific application, e.g. floods like in this case.  "specific structure of the GDL config file" indeed ...  We may need a structure of YAML files as a function of application that a template could make use of.